### PR TITLE
feat: add codecov bash uploader

### PR DESCRIPTION
**Pull request by AlekSi**
Allows us to not download it on each build.

Signed-off-by: Alexey Palazhchenko <alexey.palazhchenko@gmail.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,4 @@
 ARG DOCKER=docker:19.03.13
-ARG GOLANG=golang:1.14-alpine
```
Comment (AlekSi) [inline Dockerfile:2]: Unused?

File: Dockerfile [hunk 2]
```diff
@@ -30,16 +29,17 @@ RUN apk add --update --no-cache \
   openssh-client \
   openssl-dev \
   ovmf \
+  perl-utils \
```
Comment (AlekSi) [inline Dockerfile:32]: Added it for shasum, sorted the rest

**Review by Unix4ever — APPROVED** [review]

**Review by smira — APPROVED** [review]

**Comment by AlekSi** [comment]
https://github.com/talos-systems/build-container/pull/40

**Pull request merged**